src/EventStore.Core.XUnit.Tests/Metrics/ProcessMetricsTests.cs

```diff
@@ -228,7 +228,7 @@ public class ProcessMetricsTests : IDisposable {
 		Assert.Collection(
 			_longListener.RetrieveMeasurements("eventstore-gc-generation-size-bytes"),
 			m => {
-				Assert.True(m.Value > 0);
+				Assert.True(m.Value >= 0);
```

Comment:
It's causing flaky results, not sure how to force it to always return a value larger than zero.

Reply:
Like here: https://github.com/EventStore/EventStore/actions/runs/6732529432

Reply:
i suppose it could be 0 if a gen0 collection has run and no further allocations have been made... we are still checking that the other generations are >0 in the test so i think this is fine.